transifex/native/django/management/utils/invalidate.py

```diff
@@ -31,8 +31,8 @@ def handle(self, *args, **options):
 
         try:
             if 200 <= status_code < 300:
-                # {"count":0}
-                count = response_content.get('count')
+                # {"data": { "count":0 }}
+                count = response_content.get('data').get('count')
```

Comment:
if `response.content` doesn't have a `data` key, you end up doing `None.get('count')` which will raise an `AttributeError`

Solution:

```diff
-response_content.get('data').get('count')
+response_content.get('data', {}).get('count', 0)
```

Reply:
This is and similar cases are intentionally left this way, in order for an exception to be raised. In that case, an except block will display the raw output.